packages/component/src/hooks/useRenderEmoji.js

```diff
@@ -0,0 +1,23 @@
+import useEmojiFromStyles from './useEmojiFromStyles';
+import { useTextBoxValue } from '../SendBox/TextBox';
+import { useCallback } from 'react';
+
+export default function useRenderEmoji() {
```

Comment:
This hook, right now is:

- Read value from SendBox/TextBox
- Return the converted value

Few things:
1. It is not in a pair
   - Pair means, either
      1. Get from text box, set to text box
      1. Get from arg, return the value
1. If this is related to `SendBox/TextBox.js` only, put this hook under the `SendBox/TextBox.js`
   - Think about this: if the dev want to rewrite the text box, do we allow them to use this function?
   - Since you are grabbing the value via `useTextBoxValue` from `SendBox/TextBox.js`, this hook is "very specific to our implementation of TextBox"

I would recommend making this hook universal to any implementations of send box, including those written by developers.